app/src/main/java/org/mozilla/fenix/settings/creditcards/view/CreditCardItemViewHolder.kt

```diff
@@ -26,6 +26,9 @@ class CreditCardItemViewHolder(
     fun bind(creditCard: CreditCard) {
         credit_card_logo.setImageResource(creditCard.cardType.creditCardIssuerNetwork().icon)
 
+        credit_card_network.text = creditCard.cardType.creditCardIssuerNetwork().name
+            .capitalize(Locale.getDefault())
```

Comment:
This won't work for "amex", which should either be "AMEX" or "American Express".